client/components/editor/index.js

```diff
@@ -27,6 +27,14 @@ const plugins = [
   listPlugin
 ];
 
+const serialiseValue = value => {
+  const jsonVal = JSON.stringify(value.toJSON());
+  if (jsonVal === JSON.stringify(Value.fromJSON(initialValue('')))) {
```

Comment:
I _think_ this is ok as:
```suggestion
  if (jsonVal === initialValue('')) {
```